MANIFEST.in

```diff
@@ -15,6 +15,7 @@ recursive-include docs *.rst
 recursive-include docs Makefile
 recursive-include fastavro *.pyx
 recursive-include fastavro *.pyi
+recursive-include fastavro *.py
```

Comment:
I was getting this error, and used a given suggestion, though not sure if it's a proper solution.
```
lists of files in version control and sdist do not match!
missing from sdist:
  fastavro/repository/__init__.py
  fastavro/repository/base.py
  fastavro/repository/flat_dict.py
suggested MANIFEST.in rules:
  recursive-include fastavro *.py
```